SPECS/WALinuxAgent/WALinuxAgent.spec

```
@@ -39,6 +40,7 @@ images that are built to run in the Windows Azure environment.
 
 %prep
 %setup -q -n %{name}-%{version}
+%patch0 -p1
```

I think we should leverage `autosetup`
```suggestion
%autosetup -n %{name}-%{version} -p1
```